src/index.js

```diff
@@ -177,7 +177,107 @@ const voicemailButtonElement = document.getElementById(ELEMENT_IDS.voicemailButt
 //
 
 // HELPERS
+const renderFooter = () => {
+    // FAVOURITES_BUTTON_ELEMENT
+    favouritesButtonElement.onclick = () => {
+        renderFavourites()
+    }
+
+    // RECENTS_BUTTON_ELEMENT
+    recentsButtonElement.onclick = () => {
+        renderRecents()
+    }
+
+    // CONTACTS_BUTTON_ELEMENT
+    contactsButtonElement.onclick = () => {
+        document.getElementById(ELEMENT_IDS.contactsButton).disabled = true
+        document.getElementById(ELEMENT_IDS.keypadButton).disabled = false
+
+        renderContacts()
+    }
+
+    // KEYPAD_BUTTON_ELEMENT
+    keypadButtonElement.onclick = () => {
+        renderKeypad()
+    }
+
+    // VOICEMAIL_BUTTON_ELEMENT
+    voicemailButtonElement.onclick = () => {
+        renderVoicemail()
+    }
+}
+
+const renderInCallButtons = () => {
+    addNumberElement.innerText = INNER_TEXTS.addNumberElementInCall
+
+    document.querySelectorAll(`.${CLASS_NAMES.dialButton}, footer, #${ELEMENT_IDS.callDeleteWrapper}`).forEach((oldElement) => oldElement.remove())
+
+    IN_CALL_BUTTONS_ARRAY.forEach((button) => {
+        const inCallButtonElement = document.createElement("button")
+        inCallButtonElement.className = CLASS_NAMES.dialButton
+        inCallButtonElement.innerText = button.value
+
+        dialButtonPadElement.appendChild(inCallButtonElement)
+    })
+
+    const endButtonElementWrapper = document.createElement("div")
+    endButtonElementWrapper.id = ELEMENT_IDS.endButtonElementWrapper
+
+    const endButtonElement = document.createElement("button")
+    endButtonElement.id = ELEMENT_IDS.endButton
+    endButtonElement.className = CLASS_NAMES.dialButton
+    endButtonElement.onclick = () => {
+        displayElement.innerText = INNER_TEXTS.displayElement
+
+        document.querySelectorAll(`.${CLASS_NAMES.dialButton}, #${ELEMENT_IDS.endButtonElementWrapper}`).forEach((oldElement) => oldElement.remove())
+
+        callDeleteWrapperElement.append(callButtonElement, deleteButtonElement)
+        mainElement.appendChild(callDeleteWrapperElement)
+        appElement.appendChild(footerElement)
+
+        renderDisplay()
+        renderApp()
```

Comment:
This is not correct.


When you are rendernig an app, inside `renderApp` you should only select which view you want to see, so for example if you decide that after opening an app contacts should be visible, then you should call `renderContacts` inside `renderApp`